src/Controls/src/Core/Application.cs

```diff
@@ -348,7 +348,9 @@ public void SendOnAppLinkRequestReceived(Uri uri)
 
 		internal void SendResume()
 		{
+			if (Current is null)
```

Comment:
I'd do `if (Current != this)`, but we don't really handle multiple Application, so 👍